locations/exporters/geoparquet.py

```diff
@@ -0,0 +1,27 @@
+import geopandas
+from scrapy.exporters import BaseItemExporter
+
+from locations.exporters.geojson import item_to_geojson_feature
+
+
+class GeoparquetExporter(BaseItemExporter):
+    def __init__(self, file, **kwargs):
+        super().__init__(**kwargs)
+        self.first_item = True
+        self.file = file
+        self.features = []
+
+    def export_item(self, item):
+        if self.first_item:
+            # TODO Figure out a way to attach dataset attributes to the parquet file.
+            # self.write_dataset_attributes_table(item)
+            self.first_item = False
+
+        self.features.append(item_to_geojson_feature(item))
+
+    def finish_exporting(self):
+        # Create a GeoDataFrame from the features
+        gdf = geopandas.GeoDataFrame.from_features(self.features)
```

Comment:
This will probably need to be written in batches. I used an extreme example (lots of POIs, big json in memory too), but I got close to 6GB of ram used for the single spider.

```python
from locations.storefinders.alltheplaces import AllThePlacesSpider


class TestSpider(AllThePlacesSpider):
    name = "test_3"
    start_urls = ["https://alltheplaces-data.openaddresses.io/runs/2024-06-08-13-32-01/output/visa.geojson"]
    custom_settings = {"DOWNLOAD_TIMEOUT": 60}
    no_refs = True
```